### ISSUE TITLE
internal/FE-Cannot create docker image

### ISSUE THREAD

**Issue by milanmajchrak** [dataquest-dev/dspace-angular#374]
## Problem description
`Docker images` github action has started failing: 
![image](https://github.com/dataquest-dev/dspace-angular/assets/90026355/aea36724-22f7-4729-92eb-14cc9cb85733)

Error message:
```
#16 727.8 Node.js v18.18.0"
#16 727.8 info This module is OPTIONAL, you can safely ignore this error
#16 735.1 error /app/node_modules/@parcel/watcher: Command failed.
#16 735.1 Exit code: 1
#16 735.1 Command: node-gyp-build
#16 735.1 Arguments: 
#16 735.1 Directory: /app/node_modules/@parcel/watcher
#16 735.1 Output:
#16 735.1 node:events:495
#16 735.1       throw er; // Unhandled 'error' event
#16 735.1       ^
#16 735.1 
#16 735.1 Error: spawn node-gyp ENOENT
#16 735.1     at ChildProcess._handle.onexit (node:internal/child_process:284:19)
#16 735.1     at onErrorNT (node:internal/child_process:477:16)
#16 735.1     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
#16 735.1 Emitted 'error' event on ChildProcess instance at:
#16 735.1     at ChildProcess._handle.onexit (node:internal/child_process:290:12)
#16 735.1     at onErrorNT (node:internal/child_process:477:16)
#16 735.1     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
#16 735.1   errno: -2,
#16 735.1   code: 'ENOENT',
#16 735.1   syscall: 'spawn node-gyp',
#16 735.1   path: 'node-gyp',
#16 735.1   spawnargs: [ 'rebuild' ]
#16 735.1 }
#16 735.1 
#16 735.1 Node.js v18.18.0
#16 735.1 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
#16 ERROR: process "/bin/sh -c yarn install --network-timeout 2000000" did not complete successfully: exit code: 1
```

**Comment by milanmajchrak** [comment]
When I removed `yarn.lock` file the `yarn install` command failed when I run it locally.

**Comment by MajoBerger** [comment]
Yarn.lock file contained old versions. We needed to delete yarn.lock, run yarn install with new node (v 18.18) and save it. Now everything works.

**Comment by milanmajchrak** [comment]
1. I updated Node to version 18.18
2. Generated a new `yarn.lock` -> yarn install worked
3. Suddenly there was an error about `ngx-mask`, they released a new version of the library and it destroyed project build (yarn build:prod)
4. I updated `ngx-mask` in the `package.json` and build was successful
5. I generated a new `yarn.lock` to the git, but suddenly another error has occurred - cannot yarn install in the git for the Node v16 because of `@yarnpkg/parsers` library - actual generated version by the `yarn install` into the `yarn.lock` wasn't compatible with the Node v16..
6. I updated the `@yarnpkg/parsers` version in the `yarn.lock` following the Vanilla `yarn.lock`
